pkg/cmd/importcmd/import.go

```diff
@@ -544,173 +543,6 @@ func (o *ImportOptions) SetReporter(reporter ImportReporter) {
 	o.reporter = reporter
 }
 
-// HasJenkinsfile returns  the ile name if there is a Jenkinsfile or empty string if there is not
-func (o *ImportOptions) HasJenkinsfile() (string, error) {
```

Comment:
Shouldn't the block https://github.com/jenkins-x/jx-project/blob/d780df95daf894574244a9f9f9f1e101b61c1ce8/pkg/cmd/importcmd/import.go#L384-L387 then need to be commented/deleted too?

Reply:
its not really being used yet - but I'm hoping to add back in the Jenkins import logic so if there's a Jenkinsfile we ask folks if they want to import the source into a Jenkins server

Reply:
its slightly more complex though; as you may wish to 

* ignore the Jenkinsfile completely in jx as its being used by a separate Jenkins pipeline in a separate Jenkins server 
* import the project into a Jenkins server inside jenkins x (or outside): https://jenkins-x.io/docs/v3/guides/jenkins/getting-started/
* use Jenkinsfile runner via tekton